Prep palgrave comms

Nikoleta-v3 opened the pull request.

Nikoleta-v3 committed:
remove authors name from manuscript

Nikoleta-v3 committed:
update cover letter

Nikoleta-v3 committed:
compile cover letter and main.tex

drvinceknight commented on bibliography.bib:
```diff
@@ -863,7 +862,7 @@ @misc{anarchy_data_2018
 
 @misc{auction_data_2018, 
-  author       = {N. E. Glynatsi},
+  author       = {Author},
```
lol

drvinceknight commented on cover_letter.tex:
```diff
@@ -52,14 +52,15 @@
-researchers.
+publications in the field. One such example is the work of Youngblood and Lahti:
+``A bibliometric analysis of the interdisciplinary field of cultural evolution''
+which in 2018 was published in Palgrave Communications and
```
`in Palgrave Communicatione. Youngblood and Lahti demonstrated how ...`

drvinceknight reviewed (changes requested):
Let's just break up that one sentence (it's long).

Nikoleta-v3 committed:
break down long sentence

Nikoleta-v3 committed:
Palgrave comms corrections (#103)

* define abbreviation the first time and use afterwards

* move things around

* add TODO in abstract

* tweaks introduction

More specifically:

- the sentence regarding the novelty of the work has been moved here
- tweak the references to the sections, as the new sections are
Methods, Results, Conclusion.

* move introduction of LDA and co-authorship network before respective sections

* add todo: change title

* draft research topics section

* add todo for software reference

* address comments on methodology

These include the following changes:

- add details on the data collection process
- add a diagram to accompany these details
- move the preliminary analysis here
- tweak closing sentence

* compile document

* re run network analysis notebook and re generate measures

* add to gitignore

* Finalise title and introduction

* Finalise research topics section

* Finalise co-authorship network section

* Closes #98 #101

* correct grammar mistake pointed out by the reviewer

* Finalise conclusion

* Apply suggestions from VK

Co-authored-by: Vince Knight <vincent.knight@gmail.com>

* Update main.tex

Co-authored-by: Vince Knight <vincent.knight@gmail.com>

* replace additions with supplementing

* tweak inaccurate sentence

* apply VK suggestition

* Palgrave comms response (#104)

* add marked up pdf

I will generate this again. I just wanted to see if it compiles
correctly.

* draft response to reviewers

* Update response/main.tex

Co-authored-by: Vince Knight <vincent.knight@gmail.com>

Co-authored-by: Vince Knight <vincent.knight@gmail.com>

* Update main.tex

Co-authored-by: Vince Knight <vincent.knight@gmail.com>

* VK addresses last todo

Co-authored-by: Vince Knight <vincent.knight@gmail.com>

Co-authored-by: Vince Knight <vincent.knight@gmail.com>
